lib/http-incoming.js

```diff
@@ -1,15 +1,12 @@
-import originalUrl from 'original-url';
-import { URL } from 'url';
+import { URL } from 'node:url';
```

Comment:
Do we need to import this given that its globally available?

Reply:
No. We don't need to and I don't have any other strong arguments for importing other than that I prefer to import node libraries. Makes it explicit.